### ISSUE TITLE
mjs text/javascript mimetype

### ISSUE THREAD

**Issue by benbro** [ninenines/cowlib#138]
Is it possible to update cow_mimetypes.erl to include JavaScript modules mimetype?
It is included here:
https://svn.apache.org/repos/asf/httpd/httpd/trunk/docs/conf/mime.types

```erlang
all_ext(<<"mjs">>) -> {<<"text">>, <<"javascript">>, []};
web_ext(<<"mjs">>) -> {<<"text">>, <<"javascript">>, []};
```

**Comment by essen** [comment]
Yes for the next release. In the meantime you can run `make gen` in your Cowlib copy.

**Comment by benbro** [comment]
Thanks. Do you want me to close the issue?

**Comment by essen** [comment]
No please leave open or I'll forget to do it. Cheers!

**Comment by essen** [comment]
Done in master. Closing, thanks!!
